### PR TITLE
Fix `show attributes`

### PR DESCRIPTION
**Pull request by AlanRosenthal**
`show attributes` wasn't being populated since `show_attributes()` was never called.

Also updated `show attributes` to match the color and indentation of `show services`

### CODE REVIEW

File: apps/console.py [hunk 1]
```diff
@@ -370,9 +371,13 @@ def show_services(self, services):
 
     def show_attributes(self, attributes):
         lines = []
-
         for attribute in attributes:
-            lines.append(('ansicyan', f'{attribute}\n'))
+            if type(attribute) is Service:
```
[the remark this answers is not in the document]
Comment (AlanRosenthal) [reply apps/console.py:375]: what do you think about doing something like:
`show local-service`
`show remote-service`
or 
`show service <local|remote>`

File: apps/console.py [hunk 2]
```diff
@@ -355,24 +356,30 @@ def show_services(self, services):
         lines = []
         del self.known_attributes[:]
         for service in services:
-            lines.append(('ansicyan', str(service) + '\n'))
+            lines.append(('ansicyan', f'{service}\n'))
 
             for characteristic in service.characteristics:
-                lines.append(('ansimagenta', '  ' + str(characteristic) + '\n'))
+                lines.append(('ansimagenta', '  {characteristic}\n'))
```
Comment (mogenson) [inline apps/console.py:362]: Need an `f` here, eg `f' {characteristic}\n'`.

File: apps/console.py [hunk 3]
```diff
@@ -355,24 +356,30 @@ def show_services(self, services):
...
         for attribute in attributes:
-            lines.append(('ansicyan', f'{attribute}\n'))
+            if type(attribute) is Service:
+                lines.append(('ansicyan', f'{attribute}\n'))
+            elif type(attribute) in (Characteristic, CharacteristicDeclaration):
+                lines.append(('ansimagenta', f'  {attribute}\n'))
+            elif type(attribute) is Descriptor:
+                lines.append(('ansigreen', f'    {attribute}\n'))
+            else:
+                lines.append(('ansiyellow', f"{attribute}"))
```
Comment (mogenson) [inline apps/console.py:382]: nit: mixed single and double quotes in this section

Comment (AlanRosenthal) [reply apps/console.py]: applied black to this section

File: apps/console.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (barbibulle) [reply apps/console.py:377]: Yes, that's a good way to handle it.

**Review by barbibulle — APPROVED** [review]

**Review by barbibulle — APPROVED** [review]

**Pull request merged**